### DISCUSSION TITLE
For Attach request, allow user to key in process name instead of PID

### DISCUSSION THREAD

**Discussion by nurfitri97** [WebFreak001/code-debug#423] [Ideas]
For attach request , the PID is very malleable and will be different each time the apps is launch. Suggest to develop a new field with the name as `processName` as an alternate to existing `target` field. Thus from the new field we can use `ps aux | grep processName ` value to get the PID and attach to it.

**Comment by GitMensch** [comment]
The way it is expected to work is to show an input field that lists the processes, possibly with pre-setting a filter - and if there's only one entry it could be used directly. 

In any case that must be implemented using typescript, not system processes. 
You're very welcome to create a PR implementing that!
